extensions/wikia/ArticleVideo/scripts/jwplayer-onscroll.js

```diff
@@ -0,0 +1,92 @@
+define('wikia.articleVideo.jwPlayerOnScroll', ['wikia.onScroll'], function (onScroll) {
+	return function (playerInstance, $featuredVideo, $playerContainer) {
+		var videoCollapsed = false,
+			collapsingDisabled = false;
+
+		// TODO move to separated module
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)